File: RockitCompiler/benchmarks/bench_split.js
// Build 100-column CSV line
const parts = [];
for (let i = 0; i < 100; i++) parts.push("f" + i);
const line = parts.join(",");

let count = 0;
for (let i = 0; i < 500000; i++) {
    count = line.split(",").length;
}
console.log(count);

File: RockitCompiler/benchmarks/bench_split.go
package main

import (
	"fmt"
	"strings"
	"strconv"
)

func main() {
	// Build 100-column CSV line
	parts := make([]string, 100)
	for i := 0; i < 100; i++ {
		parts[i] = "f" + strconv.Itoa(i)
	}
	line := strings.Join(parts, ",")

	count := 0
	for i := 0; i < 500000; i++ {
		fields := strings.Split(line, ",")
		count = len(fields)
	}
	fmt.Println(count)
}
